src/test/java/study/SetCollectionTest.java

```diff
@@ -0,0 +1,59 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트를 구현한다.")
+    @Test
+    void checkSizeOfSet() {
+        // given
+        final int expected = 3;
+
+        // when
+        final int actual = this.numbers.size();
+
+        // then
+        assertThat(actual).isEqualTo(expected);
```

Comment:
size를 꺼내서 비교하는것말고 assertj 의 api 를 사용할수도 있어요!

```suggestion
        assertThat(numbers).hasSize(3);
```

Reply:
오 훨씬 명시적이고 좋아보입니다! 감사합니다 ㅎㅎ